.clang-format

```diff
@@ -61,9 +61,11 @@ SpacesInAngles:  false
 SpaceInEmptyParentheses: false
 SpacesInCStyleCastParentheses: false
 SpaceAfterCStyleCast: false
-SpacesInContainerLiterals: true
+SpacesInContainerLiterals: false
```

Comment:
I honestly thought clang-format was broken because it kept adding these annoying spaces, but it turns out I had this wonky setting turned on the entire time. :angry:

Reply:
lol.. :D